Assets/LeapMotion/Core/Scripts/Attributes/ImplementsInterface.cs

```diff
@@ -45,28 +73,48 @@ public class ImplementsInterfaceAttribute : CombinablePropertyAttribute,
                                                          property.objectReferenceValue);
 
         if (implementingObject == null) {
-          Debug.LogError(property.objectReferenceValue.GetType().Name + " does not implement " + type.Name);
+          Debug.LogError(property.objectReferenceValue.GetType().Name + " does not implement " + _type.Name);
```

Comment:
Is the _type variable safe to use here?  Can't that be null in certain situations?

Reply:
oops. I'm going to refactor this script a bit, the way I set up the type options is very fragile.